### ISSUE TITLE
The Input Map window and GDscript accepts action names with spaces, but they won't work in game.

### ISSUE THREAD

**Issue by Ace-Dragon** [godotengine/godot#17322]
In Godot 3.0.2

I recently discovered that action names containing spaces will not be acted upon when testing the game in Godot (you will need one or more underscores). An example is 
**Input.is_action_pressed('my action') vs. Input.is_action_pressed('my _action').**

Now it might be more of a usability issue than anything, because having spaces instead of an underscore produces no errors in the game or in the UI when it is being set up (instead, the function will quietly choose to not execute with no apparent reason why). This threw me off for around 25 minutes until I realized the problem was the space.

In short, if Godot does not want to have the user create actions with spaces in the name, the UI should indicate that being the case.

**Comment by Hinsbart** [comment]
Actions with spaces actually seem to work when you edit project.godot and put quotes around them. So I think instead of disallowing them we should find a way to save them with quotes.
